crypto/asn1/a_mbstr.c

```diff
@@ -342,54 +341,8 @@ static int cpy_utf8(unsigned long value, void *arg)
     return 1;
 }
 
-/* Return 1 if the character is permitted in a PrintableString */
-static int is_printable(unsigned long value)
-{
-    int ch;
-    if (value > 0x7f)
-        return 0;
```

Comment:
> I still don't think it necessary.

Correct, it's not *necessary*. That's why I said "it [comparison] is not significant". Question rather is if keeping it *anyway* improves readability/auditability. I argue that it would be an improvement if reader of the core doesn't have to wrestle with questions like what is prototype for this mimic-one-but-not-standard function,  what is implicit cast, is it safe cast under the circumstances, etc.